src/wazuh_modules/agent_upgrade/manager/wm_agent_upgrade_validate.c

```diff
@@ -25,7 +25,7 @@
  * @retval WM_UPGRADE_VERSION_SAME_MANAGER
  * @retval WM_UPGRADE_GLOBAL_DB_FAILURE
  * */
-static int wm_agent_upgrade_validate_non_custom_version(const char *agent_version, const wm_agent_info *agent_info, wm_upgrade_task *task);
+static int wm_agent_upgrade_validate_non_custom_version(const char *agent_version, const wm_agent_info *agent_info, wm_upgrade_task *task, wm_manager_configs manager_configs);
```

Comment:
Add the new parameter to the description

Reply:
Since `manager_configs `is used as read-only it could be defined as const. It is also better to use a pointer, to avoid creating a copy of the structure.
`const wm_manager_configs* manager_configs`